README.md

```diff
@@ -1,67 +1,51 @@
+| foxglove-schemas-protobuf   | [![pypi version](https://shields.io/pypi/v/foxglove-schemas-protobuf)](https://pypi.org/project/foxglove-schemas-protobuf/)     | Compiled protocol buffers                                                                                                                    |
+| foxglove-schemas-flatbuffer | [![pypi version](https://shields.io/pypi/v/foxglove-schemas-flatbuffer)](https://pypi.org/project/foxglove-schemas-flatbuffer/) | Compiled flatbuffers                                                                                                                         |
 
-These schemas can be used in [MCAP](https://github.com/foxglove/mcap) files or [Foxglove WebSocket](https://github.com/foxglove/ws-protocol) servers to take advantage of Foxglove's visualizations.
+### C++
 
-## License
+Note: C++ implementation currently lives in [foxglove/ws-protocol](https://github.com/foxglove/ws-protocol/tree/main/cpp).
 
-@foxglove/schemas is licensed under the [MIT License](https://opensource.org/licenses/MIT).
+| Package            | Version | Description                                                                                                                            |
+| ------------------ | ------- | -------------------------------------------------------------------------------------------------------------------------------------- |
+| foxglove-websocket | N/A     | C++ implementation of the WebSocket protocol<br/>Source: [foxglove/ws-protocol](https://github.com/foxglove/ws-protocol/tree/main/cpp) |
 
-## Development
+### Rust
 
-Corepack is required for development.
+Note: Rust support is alpha quality and not recommended for production use.
 
-```sh
-corepack enable
-```
+| Package  | Version                                                                                         | Description |
+| -------- | ----------------------------------------------------------------------------------------------- | ----------- |
+| foxglove | [![crates.io version](https://shields.io/crates/v/foxglove)](https://crates.io/crates/foxglove) | Rust SDK    |
 
-The schema definitions are in [internal/schemas.ts](internal/schemas.ts).
+### ROS
 
-After editing the schemas, re-generate the language-specific definitions by running `yarn update-generated-files`.
+| Package       | Version                                                                                                      | Description                                      |
+| ------------- | ------------------------------------------------------------------------------------------------------------ | ------------------------------------------------ |
+| foxglove_msgs | [![ROS version](https://img.shields.io/ros/v/rolling/foxglove_msgs)](https://index.ros.org/p/foxglove_msgs/) | Foxglove message definitions for ROS 1 and ROS 2 |
 
-### Release instructions
+### TypeScript
```

Comment:
This is not the "sdk" but a schemas package. Why isn't there a reference to the ws-protocol repo like you did for c++?

Reply:
Will add links to ws-protocol packages